src/main/java/com/pnu/ordermanagementapp/product/ProductDbAdapterImpl.java

```diff
@@ -17,38 +17,42 @@
 
     private ProductDao productDao;
 
-    private static int pageSize = 10;
+    private static int PAGE_SIZE = 10;
 
-    private static String sortByAttribute = "name";
+    private static Sort SORT = Sort.by("active").descending().and(Sort.by("name").ascending());
 
-    private static Sort.Direction sortDirection = Sort.Direction.ASC;
+    private Pageable pageable;
 
     @Autowired
     public ProductDbAdapterImpl(ProductDao productDao) {
         this.productDao = productDao;
+        this.pageable = PageRequest.of(0, PAGE_SIZE, SORT);
```

Comment:
Do not reassign fields like this.
It works like global variables now, which is always bad.
Just create it in the place of usage.